src/ngraph_api.cc

```diff
@@ -61,6 +64,14 @@ extern bool ngraph_get_currently_set_backend_name(char** backend) {
 void ngraph_start_logging_placement() { StartLoggingPlacement(); }
 void ngraph_stop_logging_placement() { StopLoggingPlacement(); }
 bool ngraph_is_logging_placement() { return IsLoggingPlacement(); }
+
+extern void ngraph_set_disabled_ops(const char* op_type_list) {
+  disabled_op_types = std::string(op_type_list);
```

Comment:
Add clean up here, conversion to a set of strings, etc.

Reply:
Made some changes in API (C vs CPP). Also added some new code in grappler part, which does the following:

current: add identityN to all fetch nodes
now: add identityN to all fetch nodes that are not disabled. No point in adding IdN to disabled nodes, since they will not be encapsulated anyway